src/sentry/similarity/index.py

```diff
@@ -273,8 +243,30 @@ def import_(self, scope, items, timestamp=None):
         for idx, key, data in items:
             arguments.extend([idx, key, data])
 
-        return index(
-            self._get_connection(scope),
-            [],
-            arguments,
-        )
+        return self.__index(scope, arguments)
+
+
+class NoopIndex(object):
```

Comment:
Conventionally we've used the `Dummy` prefix for this, and it would probably make sense to maintain consistency here:

https://github.com/getsentry/sentry/blob/da4199ac5787c2997ffdd53cd92db43990f3aea1/src/sentry/digests/backends/dummy.py#L8
https://github.com/getsentry/sentry/blob/da4199ac5787c2997ffdd53cd92db43990f3aea1/src/sentry/tasks/reports.py#L431
https://github.com/getsentry/sentry/blob/da4199ac5787c2997ffdd53cd92db43990f3aea1/src/sentry/metrics/dummy.py#L8

… etc.

Eventually we should split this into `similarity.index.{dummy,redis}` but I'd rather not do that right now because that's going ot make any conflicts with GH-5979 even worse.

Reply:
f0d0b3cd161b711cc5dd25fda0c2c0feee28b80f